content/swiftui/concepts/views/terms/table/table.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: SwiftUI table
+Description: Table view in SwiftUI
+Subjects: Mobile Development, iOS application, SwiftUI
+Tags: SwiftUI, iOS, table, view
+
+---
+
+# SWIFTUI - LIST
+
+## **A container that presents rows of data arranged in one or more columns, optionally providing the ability to select one or more members**
+
+You commonly create tables from collections of data.
+If there are more rows than can fit in the available space,  table provides vertical scrolling automatically (on macOS, the table also provides horizontal scrolling if there are more columns than can fit in the width of the view)
+
+>struct Product: Identifiable {
```

Comment:
Enclose the code example in a Markdown code block. Precede the code block with an `## Example` header and a sentence describing what the example does.  It would be good to provide an image showing what the output of this code looks like.